apps/time/src/widgets/sugang/ui/SugangTableView.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { dummyData } from '@/widgets/sugang/data';
+import { TABLE_HEADERS } from '@/widgets/sugang/model';
+import { TableData, TableNameKey } from '@/widgets/sugang/types';
+
+export default function SugangTableView({
+  tableName,
+}: {
+  tableName: TableNameKey;
+}) {
```

Comment:
```suggestion
interface SugangTableViewProps {
  tableName: TableNameKey;
}

export default function SugangTableView({ tableName }: Props) {
```
props가 많지 않지만 `interface`를 사용하는 것이 추후 코드 관리에 더 편하지 않을까 싶어요!